dali/test/python/test_pipeline.py

```diff
@@ -1206,12 +1217,12 @@ def check_duplicated_outs_pipeline(first_device, second_device):
     assert len(out) == 4
     for i in range(batch_size):
         assert isinstance(out[3][0], dali.backend_impl.TensorGPU) or first_device == "cpu"
-        out1 = out[0].as_cpu().at(i) if isinstance(out[0][0], dali.backend_impl.TensorGPU) else out[0].at(i)
-        out2 = out[1].as_cpu().at(i) if isinstance(out[1][0], dali.backend_impl.TensorGPU) else out[1].at(i)
-        out3 = out[2].as_cpu().at(i) if isinstance(out[2][0], dali.backend_impl.TensorGPU) else out[2].at(i)
+        out1 = as_array(out[0].at(i))
+        out2 = as_array(out[1].at(i))
+        out3 = as_array(out[2].at(i))
```

Comment:
When using as_array, you should use `out[0][i]`
```suggestion
        out1 = as_array(out[0][i])
        out2 = as_array(out[1][i])
        out3 = as_array(out[2][i])
```

Reply:
Sure, thanks!